feat(s3): rearrange content MTA-3637 (#2109)
scaleway/docs-content: storage/object/how-to/enable-bucket-versioning.mdx

1. Click **Object Storage** in the **Storage** section of the side menu. A list of your buckets displays.
2. Click the name of your selected bucket.
3. Click the **Bucket Settings** tab.
    <Lightbox src="scaleway-bucket-settings-tab.webp" alt="" />
3. Scroll down and click **Enable versioning** under **Bucket Versioning**.
    <Lightbox src="scaleway-enable-versioning.webp" alt="" />

  A pop-up appears to confirm versioning activation.
4. Click **Enable bucket versioning** if you agree and wish to proceed.
    <Lightbox src="scaleway-versioning-pop-up.webp" alt="" />

## How to suspend versioning

1. Click **Object Storage** in the **Storage** section of the side menu. A list of your buckets displays.
2. Click the name of your selected bucket.
3. Click the **Bucket Settings** tab.
    <Lightbox src="scaleway-bucket-settings-tab.webp" alt="" />
4. Scroll down and click **Disable versioning** under **Bucket Versioning**.

>    A pop-up appears to disclaim the following message:

Reviewer: ```suggestion
   A pop-up appears with the following message:
```
